Pesticide/Classes/DebugTableController.swift

```diff
@@ -0,0 +1,281 @@
+//
+//  DebugTableController.swift
+//  debugdrawer
+//
+//  Created by Abraham Hunt on 11/21/14.
+//  Copyright (c) 2014 Rocketmade. All rights reserved.
+//
+
+import UIKit
+import CocoaLumberjack
+
+class SectionInfo: NSObject {
+    var rowObjects = Array<RowControl>()
+}
+
+class DebugTableController: UITableViewController, UIPickerViewDelegate, UIPickerViewDataSource, UITextFieldDelegate {
+
+    var sectionObjects = [SectionInfo()]
+    let dropDownPicker = UIPickerView()
+    let consoleView = UITextView(frame: CGRect(x: 0, y: 0, width: 320, height: 200))
+    var currentField : UITextField?
+     let keyboardDoneButtonView = UIToolbar()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.tableView.backgroundColor = UIColor.darkGray
+        self.dropDownPicker.delegate = self
+        self.dropDownPicker.dataSource = self
+        self.consoleView.isEditable = false
+        self.consoleView.backgroundColor = UIColor.darkGray
+        self.consoleView.textContainerInset = UIEdgeInsetsMake(25, 0, 0, 0)
+        self.consoleView.textColor = UIColor.white
+        self.tableView.keyboardDismissMode = UIScrollViewKeyboardDismissMode.interactive
+        let cellIds : Array<ControlType> = [.Switch, .Slider, .Button, .TextInput, .Label, .Header]
+        for type in cellIds {
+            self.tableView.register(UINib(nibName: type.rawValue, bundle: Bundle(for: DebugTableController.self)), forCellReuseIdentifier:type.rawValue)
+        }
+        self.readCurrentLog()
+        self.tableView.tableHeaderView = self.consoleView
+
+        keyboardDoneButtonView.barStyle = UIBarStyle.default
+        keyboardDoneButtonView.isTranslucent = false
+        keyboardDoneButtonView.tintColor = nil
+        keyboardDoneButtonView.sizeToFit()
+
+        let doneButton = UIBarButtonItem(title: "Done", style: UIBarButtonItemStyle.done, target: self, action: #selector(DebugTableController.pickerViewDone))
```

Comment:
This comment is not removed by recet changes, but I went through all files and removed self reference where it wasn't necessary.